usecases/schema/add.go

```diff
@@ -244,6 +245,34 @@ func (m *Manager) setPropertyDefaultTokenization(prop *models.Property) {
 	}
 }
 
+func (m *Manager) setPropertyDefaultIndexing(prop *models.Property) {
+	// if IndexInverted is set but IndexFilterable and IndexSearchable are not
+	// migrate IndexInverted later.
+	if prop.IndexInverted != nil &&
+		prop.IndexFilterable == nil &&
+		prop.IndexSearchable == nil {
+		return
+	}
+
+	vTrue := true
+	if prop.IndexFilterable == nil {
+		prop.IndexFilterable = &vTrue
+	}
+	if prop.IndexSearchable == nil {
+		switch dataType, _ := schema.AsPrimitive(prop.DataType); dataType {
+		case schema.DataTypeString, schema.DataTypeStringArray:
+			// string/string[] are migrated to text/text[] later,
+			// at this point they are still valid data types, therefore should be handled here
+			fallthrough
```

Comment:
I would replace fall through with `prop.IndexSearchable = &vTrue` for better readability